### ISSUE TITLE
Reproduce issue.

### ISSUE THREAD

**Issue by leoshine** [VainF/DeepLabV3Plus-Pytorch#9]
With the default training setting of this code, I train "deeplabv3plus_resnet101" model on voc12.
The best mIOU I can get is 0.763, whereas the provided corresponding model can score 0.783.

**Comment by VainF** [comment]
Hi @leoshine , which dataset was used for yout training? voc12 or voc12aug? The pretrained models are trained on voc12aug which has more training data than the standard voc12.

**Comment by leoshine** [comment]
Yes. I use voc12aug.  I just turn off the visualization. 
Nothing is changed:

python main.py --model deeplabv3plus_resnet101   --gpu_id 0,1,2,3 --year 2012_aug --crop_val --lr 0.01 --crop_size 513 --batch_size 16 --output_stride 16

**Comment by VainF** [comment]
https://github.com/VainF/DeepLabV3Plus-Pytorch/issues/5#issuecomment-609459776. 
This issue is caused by the batchnorm. There is no sync BN in this repo and training with multiple GPUs will simply split the batch and distribute them to each device. So each GPU only gets 16 / 4 = 4 samples and there is no communication between different devices (this is harmful for BN). It is recommended to train the model on a single GPU or use some third-party SyncBN to replace the default one.

**Comment by leoshine** [comment]
Thanks for the information.
I've tried this version of SyncBN: https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
And I can now reproduce the mIOU=78.3%. 
Great work. Thanks.

**Comment by Haoteng0612** [comment]
> Yes. I use voc12aug. I just turn off the visualization.
> Nothing is changed:
> 
> python main.py --model deeplabv3plus_resnet101 --gpu_id 0,1,2,3 --year 2012_aug --crop_val --lr 0.01 --crop_size 513 --batch_size 16 --output_stride 16
> 
> B.t.w. I downloaded the released model (DeepLabV3Plus-ResNet101) and tested it. The mIOU can exactly reach 0.783.

How do you test and get 0.783. Are there any tricks when testing since I can not get this point with the released model.
